vms/vms/settings.py

```diff
@@ -61,8 +62,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'vms',
-        'USER': 'myuser',
-        'PASSWORD': 'mypassword',
+        'USER': 'vmsadmin',
+        'PASSWORD': '0xdeadbeef',
         'HOST': 'localhost',
     }
```

Comment:
@tapasweni-pathak I have made a mistake here.....shouldn't have pushed changed credentials. I will change that